packages/wallet-apis/src/client.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  createWalletClient,
+  type Chain,
+  type JsonRpcAccount,
+  type LocalAccount,
+} from "viem";
+import { smartWalletActions } from "./decorators/smartWalletActions.js";
+import type { AlchemyTransport } from "@alchemy/common";
+
+export type CreateSmartWalletClientParams = {
+  account: LocalAccount | JsonRpcAccount;
+  transport: AlchemyTransport;
+  chain: Chain;
+  policyIds?: string[];
+};
+
+/**
+ * Creates a smart wallet client with wallet API actions.
+ *
+ * @param {CreateSmartWalletClientParams} params - Parameters for creating the smart wallet client.
+ * @param {LocalAccount | JsonRpcAccount} params.account - The account to use for signing.
+ * @param {AlchemyTransport} params.transport - The transport to use for RPC calls.
+ * @param {Chain} params.chain - The blockchain network to connect to.
+ * @param {string[]} params.policyIds - Optional policy IDs for paymaster service.
+ * @returns {WalletClient} A wallet client extended with smart wallet actions.
+ */
+// TODO(v5): do we actually want to provide this? or just instruct consumers to extend their own wallet client w/ the actions?
+export const createSmartWalletClient = ({
+  account,
+  transport,
+  chain,
+  policyIds,
+}: CreateSmartWalletClientParams) => {
+  return createWalletClient({
+    account,
```

Comment:
@adamegyed how do you feel about the use of creating a wallet client & extending it w/ `smartWalletActions`, where the `account` on the client is the signer?

in v4, the signer was a separate param from the client. i *think* this makes sense to better align w/ viem, but i'm second guessing myself a little bit.

cc @blakecduncan if you have any thoughts

Reply:
i thought about this a bit more & have some more thoughts now. might be worthing writing up some quick examples and weighing pros & cons. (will plan on doing that in the morning.)